mlflow/store/tracking/file_store.py

```diff
@@ -283,12 +283,12 @@ def delete_experiment(self, experiment_id):
     def restore_experiment(self, experiment_id):
         experiment_dir = self._get_experiment_path(experiment_id, ViewType.DELETED_ONLY)
         if experiment_dir is None:
-            raise MlflowException("Could not find deleted experiment with ID %d" % experiment_id,
+            raise MlflowException("Could not find deleted experiment with ID %s" % experiment_id,
```

Comment:
That is a good catch. However, the REST API assume experiment ID as string. For consistency, can we do the following

- Revert back this `%s` -> `%d` change 
- explicitly cast `str(experiment_id)`